test/fixtures/rules/std_rule_cases/L063.yml

```diff
@@ -134,3 +133,16 @@ test_pass_sparksql_complex_data_types:
     rules:
       L063:
         extended_capitalisation_policy: upper
+
+test_pass_bigquery_struct_params:
```

Comment:
Think it'd be worth adding a similar failing test case?

Reply:
Will do.

Reply:
Actually, won’t do.

The point of this test case was to catch the false positive whereby the param name was being incorrectly checked by this rule. So there is no fail case as this rule shouldn’t be looking at params. This test case ensures that.

The things it SHOULD be looking at are covered by other test cases already.